harvester/harvester
# Bump Longhorn dependency to v1.4.1

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
Need to bump Longhorn dependency to v1.4.1

**Solution:**
Bump Longhorn go mod and dependency chart to v1.4.1

**Related Issue:**
https://github.com/harvester/harvester/issues/3747

**Test plan:**
- build a new ISO image based on the current PR
- verify all pods are running correctly on the `longhorn-system` namespace
- create a VM and check that LH-related features are working as expected. (e.g., snapshot, backup and volumes)
<img width="2241" alt="image" src="https://user-images.githubusercontent.com/4569037/230876962-49923967-0e5e-4194-82b9-e4ed2a712704.png">
